matflow/data/scripts/mtex/sample_orientations_CRC.m

```diff
@@ -0,0 +1,81 @@
+function sample_texture_CRC(inputs_JSON_path, outputs_HDF5_path)
```

Comment:
This seems to work anyway, but for consistency with the other MTEX scripts, can you rename this function to `sample_orientations_CRC` so it matches the file name?